stl/inc/strstream

```diff
@@ -251,7 +251,7 @@ protected:
         const auto _Seeklow  = eback();
         const auto _Seekdist = _Seekhigh - _Seeklow;
 
-        // [depr.strstreambuf.virtuals]/15 effectively says check that the result will be in range
+        // N4727 [depr.strstreambuf.virtuals]/15 effectively says check that the result will be in range

```

Comment:
This is para 16 in N4885 if you'd like to avoid referencing the "ancient" N4727. I'm not bothered enough to Require Changes.

Reply:
Thanks - multiple lines are affected, so I don't want to extend this PR more. We have #182 tracking this.